content/How to change the default datepicker on the Django admin page - Jorge Michelena.md

```diff
@@ -0,0 +1,146 @@
+When you register a model in your Django admin page it automatically matches each of the model's fields to
+a widget so you can input data and manage instances of the models easily from the Django admin panel, 
+most of the time the default widgets are enough but in some cases they may not be the most suitable for our particular needs.
+
+For instance, if one of your models has a `DateField` which represents the birth date of a user and you know that for whatever
+reason you will need to create user instances frequently from the admin panelthen the default datepicker may be very inconvenient
+if you want to input realistic data since you can't input the year in a straightforward way and would have to navigate month to month to reach a desired year.
+
+In such a case you may want to change the widget to pick the date but after doing some research may not find a clear explanation
+on how to do this, but you are on luck because not only is it pretty easy but it is also what we will be guiding through in this article! Let's begin.
```

Comment:
```suggestion
In that case, you may want to change the widget to pick the date. If after doing some research you don't find a clear explanation
on how to do this, don't worry, this is your lucky day because not only it is pretty easy but it is also what I will be guiding you through in this article! Let's begin.
```